build/docker/Dockerfile-stable

```diff
@@ -30,25 +27,20 @@ RUN apt-get update && apt-get install -q -y --fix-missing \
 RUN pip install --upgrade pip
 RUN gem install zapr
 RUN pip install zapcli
-# Install latest dev version of the python API
-RUN pip install python-owasp-zap-v2.4
```

Comment:
One advantage of leaving this after zapcli is that it makes sure the latest version of `python-owasp-zap-v2.4` is installed even if `zapcli` is a version behind for example. I try to keep the version of the dependency used by zapcli constantly up to date since there aren't usually breaking changes, but I sometimes am a bit slow, and this makes sure people can use the latest API client library in their own scripts. :)